jobs/cronjob-ldap-group-sync.yml

```diff
@@ -62,65 +82,71 @@ objects:
                 command:
                   - "/bin/bash"
                   - "-c"
-                  - "oc adm groups sync --sync-config=/etc/config/ldap-group-sync.yaml --confirm || :"
+                  - "oc adm groups sync --sync-config=/etc/config/ldap-group-sync.yaml --confirm"
                 volumeMounts:
                   - mountPath: "/etc/config"
                     name: "ldap-sync-volume"
+                  - mountPath: "/etc/secrets"
+                    name: "ldap-bind-password"
             volumes:
-              - configMap:
-                  items:
-                    - key: ldap-sync.yml
-                      path: ldap-group-sync.yaml
-                  name: ldap-config
-                name: ldap-sync-volume
+              - name: "ldap-sync-volume"
+                configMap:
+                  name: "ldap-config"
+              - name: "ldap-bind-password"
+                secret:
+                  secretName: "${LDAP_BIND_PASSWORD_SECRET}"
             restartPolicy: "Never"
             terminationGracePeriodSeconds: 30
             activeDeadlineSeconds: 500
             dnsPolicy: "ClusterFirst"
             serviceAccountName: "${JOB_SERVICE_ACCOUNT}"
             serviceAccount: "${JOB_SERVICE_ACCOUNT}"
-- kind: ClusterRole
-  apiVersion: v1
+
+- kind: "ClusterRole"
+  apiVersion: "v1"
   metadata:
-    name: ldap-group-syncer
+    name: "ldap-group-syncer"
+    namespace: "${NAMESPACE}"
     labels:
       template: "cronjob-ldap-group-sync"
   rules:
     - apiGroups:
         - ""
-        - user.openshift.io
+        - "user.openshift.io"
       resources:
-        - groups
+        - "groups"
       verbs:
-        - get
-        - list
-        - create
-        - update
-- kind: ClusterRoleBinding
-  apiVersion: v1
+        - "get"
+        - "list"
+        - "create"
+        - "update"
+
+- kind: "ClusterRoleBinding"
+  apiVersion: "v1"
   groupNames: null
   metadata:
-    name: system:ldap-group-syncers
-    labels:
```

Comment:
Is there a reason to remove the `labels` here? ... see my question above re: does a "global" one at the top level apply to all objects and that's the reason?

Reply:
I removed it because it was redundant with the global labels.